app/graphql/users/resolvers.js

```diff
@@ -0,0 +1,18 @@
+const logger = require('../../logger');
+const usersHelpers = require('../../helpers/users');
```

Comment:
I think you should approach this in a more generic manner and just do a utils or helpers file for now since in a project rather than `encoding a password` you can think of it as `encoding something` so an encode function would make more sense to me

Reply:
I understand, I'll do it